packages/epics/src/people/components/edit-person-section.tsx

```
@@ -48,15 +151,36 @@ export const EditPersonSection = ({
+  const saveChanges = async () => {
+    try {
+      const updatedProfile = await editProfile({
+        name: watch('name'),
+        surname: watch('surname'),
+        nickname: watch('nickname'),
+        leadImageUrl: watch('leadImage'),
+        description: watch('description'),
+        id: id,
+      });
+      console.log('Profile updated:', updatedProfile);
+      successfulEditCallback;
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)